frontend/packages/kubevirt-plugin/src/selectors/vm/combined.ts

```diff
@@ -0,0 +1,17 @@
+import * as _ from 'lodash';
+import {
+  VM_STATUS_IMPORTING,
+  VM_STATUS_V2V_CONVERSION_IN_PROGRESS,
+} from 'kubevirt-web-ui-components';
+
+import { VMIKind, VMKind } from '../../types/vm';
+import { isVMRunning } from './selectors';
+import { VMMultiStatus } from '../../types';
+
+const IMPORTING_STATUSES = new Set([VM_STATUS_IMPORTING, VM_STATUS_V2V_CONVERSION_IN_PROGRESS]);
+
+export const isVMImporting = (status: VMMultiStatus): boolean =>
+  status && IMPORTING_STATUSES.has(status.status);
```

Comment:
I think using array `includes` is easier here:

```ts
status && [VM_STATUS_IMPORTING, VM_STATUS_V2V_CONVERSION_IN_PROGRESS].includes(status.status)
```

Reply:
Both accomplish the same thing, but the functionality is semantically closer to set.  

I am not aware of any issue with using a set (if implemented as a global constant).

Reply:
OK, I'm fine with `Set` too.